### PR TITLE
Smarter GitHub Actions Instance Even Distribution | Skipped Rationale Generation 5 and Fixed Abductive Reasoning 11 Removing Input Sorting, Fixed DI Rationale Gen. evaluation - single 2

### PR DESCRIPTION
**Pull request by klxu03**
Applied a greedy algorithm to split_tasks for TAP tests into 20 more evenly distribute github action instances. Noticed some completing in 30 mins and others taking 5 hours. This should make it more even. 

Fixed Abductive Reasoning 11 by getting rid of sort input_fields by X coords then Y coords in extract_input_values_from_url. I think we should establish a new convention where the order of filling in is specified by the order of Answer columns, lefter ones going first and we manually specify for more fine grained control over each task. (This may cause many other tasks to fail, in which we should properly manually re-order right now)

Likewise fixed DI Rationale Gen. evaluation - single 2 same way, rearranging column order (there may be other tasks like this that my previous fix broke). This is still flaky, going to wait for TAP tests to give me an index of error

Skipping the Rationale Generation 5 task. you need to click "show questions" before the questions show up for u to solve. We can manually code this action in for this task if we want, or keep it skipped

@danyaljj @yeganehkordi

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            "name": "Python: Current File",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal",
+            "cwd": "/src/",
+            "justMyCode": true
+        }
+    ]
+}
```
Comment (danyaljj) [inline .vscode/launch.json]: This should be dropped?

**Comment by danyaljj** [comment]
One minor comment. Otherwise, good to go.

File: .vscode/launch.json (hunk 1 above)
Comment (klxu03) [reply .vscode/launch.json]: Oh yes! I'll go throw this in the gitignore

**Comment by klxu03** [comment]
> One minor comment. Otherwise, good to go.

@danyaljj was this the update you wanted?

**Review by danyaljj — APPROVED** [review]
Looks good. Merge when you're happy!

**Pull request merged**